userspace/libscap/CMakeLists.txt

```diff
@@ -39,19 +39,10 @@ if(CMAKE_SYSTEM_NAME MATCHES "Linux")
 		set(KBUILD_FLAGS "${FALCOSECURITY_LIBS_DEBUG_FLAGS}")
 	endif()
 
-    if(NOT DEFINED DRIVER_VERSION)
-        message(ERROR 
-			"No driver version set. Please either build the project from a git working directory or explicitly set the DRIVER_VERSION cmake cache entry."
-		)
-    endif()
-
-    if(NOT DEFINED DRIVER_NAME)
-        set(DRIVER_NAME "scap")
-    endif()
-
-    if(NOT DEFINED DRIVER_DEVICE_NAME)
```

Comment:
This seems to be used by scap too: https://github.com/falcosecurity/libs/blob/fe28e8306709735804e2ca1adeb0136872ebf52b/userspace/libscap/engine/kmod/scap_kmod.c#L126